pkg/registry/discovery/rest/storage_discovery.go

```diff
@@ -44,6 +45,14 @@ func (p StorageProvider) NewRESTStorage(apiResourceConfigSource serverstorage.AP
 		apiGroupInfo.VersionedResourcesStorageMap[discoveryv1alpha1.SchemeGroupVersion.Version] = storageMap
 	}
 
+	// if apiResourceConfigSource.VersionEnabled(discoveryv1beta1.SchemeGroupVersion) {
```

Comment:
what's with these comments?

Reply:
No idea why/when I did that, fixed. Thanks for catching this!